evalml/model_understanding/graphs.py

```diff
@@ -487,3 +486,57 @@ def _calculate_axis_range(arr):
     min_value = arr.min()
     margins = abs(max_value - min_value) * 0.05
     return [min_value - margins, max_value + margins]
+
+
+def _get_prediction_vs_actual_data(y_true, y_pred, outlier_threshold):
+    """Helper method to help calculate the y_true and y_pred dataframe, with a column for outliers"""
+    predictions = y_pred.reset_index(drop=True)
+    actual = y_true.reset_index(drop=True)
+    data = pd.concat([pd.Series(predictions),
+                      pd.Series(actual)], axis=1)
+    data.columns = ['prediction', 'actual']
+    data['outlier'] = np.where((abs(data['prediction'] - data['actual']) >= outlier_threshold), "#ffff00", "#0000ff")
+    return data
+
+
+def graph_prediction_vs_actual(y_true, y_pred, outlier_threshold=100):
```

Comment:
Maybe we should not provide a default value for the `outlier_threshold`? What constitutes a large difference depends on the units of the target variable which we can't know without user input.

Reply:
Mm good point. I can default it as `None` then